ha/core/controllers/pcs/service_controller.py

```diff
@@ -82,3 +100,24 @@ def status(self, service: str, nodeids: list = None) -> dict:
                 status: Succeeded, Failed, InProgress
         """
         raise HAUnimplemented("This operation is not implemented.")
+
+    def clear_resources(self, node_id: str):
+        """
+        Clear resources on node.
+
+        Args:
+            nodeid (str): Private fqdn define in conf store.
+        """
+        try:
+            resources: list = []
+            output, _, _ = self._execute.run_cmd(const.LIST_PCS_RESOURCES, check_error=False)
+            if "NO resources".lower() not in output.lower():
+                for resource in output.split("\n"):
+                    res = resource.split(":")[0]
+                    if res != "" and res not in resources:
+                        resources.append(res)
+            for resource in resources:
+                self._execute.run_cmd(f"pcs resource clear {resource} {node_id}")
```

Comment:
Use a define from ha/const.py instead of pcs command directly

Reply:
Done